rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -234,7 +242,7 @@ ln -s /tmp/pai-root/log/$APP_ID/$CONTAINER_ID/DockerContainerDebug.log $LAUNCHER
 docker pull {{{ jobData.image }}} \
   || { echo "Can not pull Docker image"; exit 1; }
 docker run --name $docker_name \
-  --rm \
+  --detach \
```

Comment:
So docker container won't be removed automatically.
Where do you clean up stopped containers?

Reply:
Line 62 in exit_handler